query-service-impl/src/test/resources/trino_service_request_handler.conf

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
We can remove this file itself. I guess we don't need them.

Reply:
removed